src/redux/selectors/resourceSelectors.ts

```diff
@@ -246,3 +247,17 @@ export const resourceNavigatorSelector = createSelector(
     return list;
   }
 );
+
+export const filteredResourcesIdsSelector = createSelector(
+  [getActiveResourceMetaMapFromState, state => state.main.resourceFilter],
+  (resources, resourceFilter) =>
+    Object.values(resources)
+      .filter(
+        r =>
+          r.kind &&
+          isResourcePassingFilter(r, resourceFilter) &&
+          !isKustomizationResource(r) &&
+          !isKustomizationPatch(r)
+      )
+      .map(r => r.id)
+);
```

Comment:
Now that Ole's PR is merged we have a new `navigatorResourcesSelector` which could be used here because it merges the active storage together with transient resources and applies the filtering:

```suggestion
export const filteredResourcesIdsSelector = createSelector(
  navigatorResourcesSelector,
  (resources) => Object.values(resources).map(r => r.id)
);
```